COPYRIGHT.txt

```diff
@@ -118,6 +118,11 @@ Copyright: 2007, Starbreeze Studios
  2014-2022, Godot Engine contributors.
 License: Expat and Zlib
 
+Files: ./servers/rendering/renderer_rd/shaders/taa_resolve.glsl
```

Comment:
Sorry to be nitpicky, but this should go below the ASSAO entry as we try to keeo these in alphabetical order.